index.ts

```diff
@@ -77,9 +95,16 @@ export const sandbox = {
     const client = await _arcFunctions.tables()
     if (seedFile) await seedDb(seedFile, dynamodbClient)
 
+    const tableStreams = inv['tables-streams']
+    const ddbStreamsClient = new DynamoDBStreamsClient({
+      region: inv.aws.region,
+      endpoint: `http://localhost:${getPort(inv)}`,
+      credentials,
+    })
+    // Init table streams for those defined
```

Comment:
Move this, and the UpdateTableCommand bit, into the `if (tableStreams)` section.

Also, note that the endpoint URL is returned by `launch()`.